en/development/routing.rst

```diff
@@ -685,7 +685,7 @@ The current prefix will be available from the controller methods through
 ``$this->request->getParam('prefix')``
 
 When using prefix routes it's important to set the ``prefix`` option, and to
-use the same camel cased format that is used in the ``prefix()`` method. Here's
+use the same camel-cased format that is used in the ``prefix()`` method. Here's
```

Comment:
```suggestion
use the sameCamelCased format that is used in the ``prefix()`` method. Here's
```

Should we be consistent everywhere? If we're going to commit to the style, let's do it, right?

Reply:
There is a Whitespace issue